sizing.tex

```diff
@@ -102,6 +103,11 @@ \section{Sizing}\label{sizing}
 over the network rather than through physical media, as originally planned,
 decoupling science from maintenance and upgrade activities to a greater extent.
 
+The external bandwidth from NCSA to the community alert brokers is baselined at 10 Gbit/sec.
```

Comment:
Use `~` or `\,` to connect numbers with units to stop them being wrapped into separate lines.